docs/dabp/guides/technical/create-resource-via-api.md

```diff
@@ -0,0 +1,148 @@
+# Creating resource via API
+Resources can get created either [through the UI](../functional/manage-resource-via-ui.md) or through the API.
+To create a resource you first need to [create a resource type via api](./create-resource-type-via-api.md).
+This page is describing how the resource API is used.
+
+## How to create a resource
+In this guide, we will assume Onegini Identity Cloud is available under the URL: `https://dabp.onegini.com`.
+The API is protected with OIDC, so you need to provide an access token to make a successful request.
+Please note that the access token must be linked to a person who has the `Add/edit/delete resource and resource types` permission 
```

Comment:
How do we do this for `non-personal requests`?
Which we now also support I guess...

Reply:
fixed - added link to page where we describe m2m calls